routes/web/project/load_balancer.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class CloverWeb
+  hash_branch(:project_prefix, "load-balancer") do |r|
+    r.get true do
+      lbs = @project.private_subnets.flat_map { |ps| ps.load_balancers_dataset.authorized(@current_user.id, "LoadBalancer:view").all }
+      @lbs = Serializers::LoadBalancer.serialize(lbs, {include_path: true})
+
+      view "networking/load_balancer/index"
+    end
+
+    r.post true do
+      Authorization.authorize(@current_user.id, "LoadBalancer:create", @project.id)
+
+      ps = PrivateSubnet.from_ubid(r.params["private-subnet-id"])
```

Comment:
Should we check if this person has permission for this private subnet?

Reply:
Makes sense, updated.